src/navigation.tpl

```diff
@@ -189,7 +189,8 @@
                   <li><a href="reference-numbering.html">Machine/unit numbering</a></li>
                   <li><a href="glossary.html">Glossary</a></li>
                   <li><a href="reference-reviewers.html">Charm Review Criteria</a></li>
-                  <li><a href="contributing.html">Contribute to the Docs!</a></li>
                 </ul>
               </li>
+               <li><a href="contributing.html">Helping with these docs</a></li>
+               <li><a href="https://github.com/juju/docs/issues/new">Report an issue with these docs</a></li>
```

Comment:
this is probably way too long and will wrap horribly. I guess you couldn't tell. What about "Report a docs issue" or "Report a docs Bug"